<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistersWidget</class>
 <widget class="QWidget" name="RegistersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>228</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <layout class="QGridLayout" name="registerGrid16">
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="5" column="1">
        <widget class="Reg16Widget" name="regPC" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Reg16Widget" name="regShadowAF" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Reg16Widget" name="regShadowBC" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="Reg16Widget" name="regAF" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="Reg16Widget" name="regDE" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="Reg16Widget" name="regSP" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="Reg16Widget" name="regIY" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Reg16Widget" name="regBC" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="Reg16Widget" name="regHL" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Reg16Widget" name="regShadowHL" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Reg16Widget" name="regShadowDE" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Reg16Widget" name="regIX" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="verticalBar1">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="flagsLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="signFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="signFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Sign flag</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">S</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="zeroFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="zeroFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Zero flag</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">Z</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="halfCarryFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="halfCarryFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Half Carry flag (carry from bit 3 to bit 4)</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">H</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="parityOverflowFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="parityOverflowFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Parity or Overflow flag</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">P/V</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="subtractFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="subtractFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Subtract flag (set if last operation was a subtraction)</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">N</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="carryFlagFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="frameLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="carryFlagLabel">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Carry flag</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 5px; padding-right: 5px</string>
            </property>
            <property name="text">
             <string notr="true">C</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="verticalBar2">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="registerGrid8">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="1" column="0">
      <widget class="Reg8Widget" name="regC" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="Reg8Widget" name="regD" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="Reg8Widget" name="regL" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="Reg8Widget" name="regB" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="Reg8Widget" name="regShadowL" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="Reg8Widget" name="regE" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="Reg8Widget" name="regH" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="Reg8Widget" name="regShadowB" native="true"/>
     </item>
     <item row="4" column="1">
      <widget class="Reg8Widget" name="regShadowH" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="Reg8Widget" name="regShadowE" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="Reg8Widget" name="regShadowD" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="Reg8Widget" name="regShadowC" native="true"/>
     </item>
     <item row="2" column="2">
      <widget class="Reg8Widget" name="regF" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="Reg8Widget" name="regA" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="Reg8Widget" name="regShadowA" native="true"/>
     </item>
     <item row="3" column="2">
      <widget class="Reg8Widget" name="regShadowF" native="true"/>
     </item>
     <item row="4" column="2">
      <widget class="Reg8Widget" name="regI" native="true"/>
     </item>
     <item row="5" column="2">
      <widget class="Reg8Widget" name="regR" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reg16Widget</class>
   <extends>QWidget</extends>
   <header>debugger/Reg16Widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Reg8Widget</class>
   <extends>QWidget</extends>
   <header>debugger/Reg8Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
